docs/source/cli_help/help.rst

```diff
@@ -0,0 +1,11 @@
+.. _help:
+
+help
+----
+Usage:
+
+.. code-block:: shell
+
+    git machete help [<command>]
+
+Prints a summary of this tool, or a detailed info on a command if defined.
```

Comment:
```suggestion
Prints a summary of this tool, or a detailed info on a command if provided.
```

`defined` suggests that for certain commands help is defined and for certain others it isn't 🤔